insights/tests/client/collection_rules/test_map_components.py

```diff
@@ -77,8 +79,11 @@ def test_get_component_by_symbolic_name():
         'ironic_conf',
         'octavia_conf',
         'partitions',
+        'rhn_entitlement_cert_xml',
         'rhn_hibernate_conf',
+        'rhn_schema_version'
```

Comment:
@Rohan21Lobo - you missed the comma here which caused the test failed.

Reply:
fixed, sorry for the typo.